ipatests/test_integration/test_host.py

```diff
@@ -0,0 +1,109 @@
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+class TestHost(IntegrationTest):
+
+    topology = 'line'
+
+    def check_host_contains_attribute(self, host, expected):
+        """
+        Helper function to test if host contains expected attribute, using both
+        ipa host-show and ldapsearch.
+        """
+        host_show_result = host.run_command(["ipa host-show", host, "--all"])
+        assert expected in host_show_result.stdout_text
+        ldapsearch_result = host.run_command(["ldapsearch -h", host])
```

Comment:
This command is browsing the whole LDAP tree, it's overkill for what you want to check. The test should define a base search, a scope and a search filter.
Search can be: fqdn=$hostfqdn,cn=computers,cn=accounts,$basedn, scope: base, filter: (objectclass=ipahost)
The test should check the value for a given attribute, not any random string output.